mopidy_local/__init__.py

```diff
@@ -27,6 +27,7 @@ def get_config_schema(self):
         schema["scan_flush_threshold"] = config.Integer(minimum=0)
         schema["scan_follow_symlinks"] = config.Boolean()
         schema["excluded_file_extensions"] = config.List(optional=True)
+        schema["included_file_extensions"] = config.List(optional=True)
```

Comment:
Maybe reorder `included` and `excluded` everywhere to always come in the same order? I think I prefer `included` to come first, without being able to express exactly why.

Reply:
Oops, yes. I originally added `included` second, since it was new, then reorganized everything to be in the `included`/`excluded` order, but missed this one.